### PR TITLE
Added SONOS to Mediaplayers

### PR DESCRIPTION
**Pull request by chrecht**

### CODE REVIEW

File: regexes/client/mediaplayers.yml [hunk 1]
```diff
@@ -65,6 +65,10 @@
   name: 'SubStream'
   version: '$1'
 
+- regex: 'Sonos/([\d\.-]+)?'
+  name: 'SONOS'
```
Comment (sanchezzzhak) [inline regexes/client/mediaplayers.yml:69]: add a test into this PR
inset test in file `device-detector/Tests/fixtures/mediaplayer.yml`

is this the detect of a Sanos client? (sonos.com)

**Review by sanchezzzhak — CHANGES_REQUESTED** [review]

File: Tests/Parser/Client/fixtures/mediaplayer.yml [hunk 2]
```diff
@@ -161,3 +161,25 @@
...
+    name: SONOS
+    version: 57.4-83220
+
+- 
+  user_agent: Linux UPnP/1.0 Sonos/61.1-83220 (ZPS3)
+  client:
+    type: mediaplayer
+    name: SONOS
+    version: 61.1-83220
+
```
Comment (sanchezzzhak) [inline Tests/Parser/Client/fixtures/mediaplayer.yml:185]: fix warning YAML LINT  (remove line 186)

**Review by sanchezzzhak — CHANGES_REQUESTED** [review]

File: regexes/client/mediaplayers.yml (hunk 1 above)
Comment (chrecht) [reply regexes/client/mediaplayers.yml:69]: Yes this detection is for SONOS Players

**Review by sanchezzzhak — APPROVED** [review]

File: Tests/Parser/Client/fixtures/mediaplayer.yml [hunk 3]
```diff
@@ -161,3 +161,24 @@
...
+- 
+  user_agent: Linux UPnP/1.0 Sonos/57.4-83220 (ZP120)
+  client:
+    type: mediaplayer
+    name: SONOS
+    version: 57.4-83220
```
Comment (sgiehl) [inline Tests/Parser/Client/fixtures/mediaplayer.yml:177]: @sanchezzzhak are we matching versions including a `-` anywhere else? 🤔

Comment (sanchezzzhak) [reply Tests/Parser/Client/fixtures/mediaplayer.yml:177]: I checked this information, there is not a single test with a version separated by a dash.

File: regexes/client/mediaplayers.yml [hunk 4]
```diff
@@ -65,6 +65,10 @@
   name: 'SubStream'
   version: '$1'
 
+- regex: 'Sonos/([\d\.-]+)?'
```
Comment (sanchezzzhak) [inline regexes/client/mediaplayers.yml:68]: remove dash 

```yml
- regex: 'Sonos/([\d\.]+)?'
```

**Review by sanchezzzhak — CHANGES_REQUESTED** [review]

**Review by sanchezzzhak — APPROVED** [review]

**Comment by sgiehl** [comment]
@chrecht  Thanks for your contribution 👍

**Pull request merged**